### PR TITLE
Generated new snapshots for version 0.33.0 as the snapshot format has changed

### PR DESCRIPTION
**Pull request by agarwal-navin**
The handles in old snapshots with relative paths are updated to be written out as absolute path in the new snapshot format. This change moves the current snapshots (before the change) to a new folder under `src_snapshots\0.33.0`.
The current snapshots are updated to the new version (0.34.0).

The FF change that updates the handles - https://github.com/microsoft/FluidFramework/pull/4934

### CODE REVIEW

**Comment by vladsud** [comment]
Any chance we can improve our test story here by keeping old files intact? We are losing coverage here, i.e. nothing in the test collateral will check old format and thus we may break support for old features that can show up in files at rest?
The ideal solution would be 
- we never update old files. They are only used for loading (i.e. starting container from).
- Each version when we do change file format, we create new versions and use them for comparison in test.

Basically, every snapshot can have versions 1, 2, ..., N, and versions [1...N] can be used for loading container from, and version N is used for comparison (writing out snapshot).

It should not take that much time to build that functionality - maybe a day or two of uninterrupted work :)

**Comment by agarwal-navin** [comment]
That is exactly what is happening :-). The change to do this went in couple of days back and I did that precisely for this change - https://github.com/microsoft/FluidFramework/pull/4934.

Here, the snapshot format changed, so the current snapshots (which were from version 0.33.0) were moved under "src_snapshots/0.33.0". And then we updated the current snapshots to the new format.

I have this change in PR that will load old snapshots from all versions 1,2,..,N-1,N, write them out and compare with N:
https://github.com/microsoft/FluidFramework/pull/4870

**Comment by agarwal-navin** [comment]
The PR that added the changes you mentioned: https://github.com/microsoft/FluidFramework/pull/4919

**Review by anthony-murphy — APPROVED** [review]
